gapis/service/service.proto

```diff
@@ -1323,18 +1323,22 @@ message ProfilingData {
         TimeWeightedAvg = 1;
       }
       int32 id = 1;
-      string name = 2;
-      string unit = 3;
-      AggregationOperator op = 4;
-      string description = 5;
-      bool select_by_default = 6;
+      uint32 counter_id = 2;  // -> Counter.id. Valid for GPU counter metrics.
+      string name = 3;
+      string unit = 4;
+      AggregationOperator op = 5;
+      string description = 6;
+      bool select_by_default = 7;
     }
 
     // Perf includes a best-guessing performance value and a confidence range.
     message Perf {
       double estimate = 1;
       double min = 2;
       double max = 3;
+      map<int32, double> estimateSamples = 4;  // {index} -> {sample weight}
+      map<int32, double> minSamples = 5;       // {index} -> {sample weight}
+      map<int32, double> minSamples = 5;       // {index} -> {sample weight}
```

Comment:
It looks like this duplicate line makes the CI builds fail.

Reply:
Oops, included this when I was desperately solving the presubmit format issues exposed by the remote kokoro result. Just fixed, thanks for noticing!